src/composables/usePool.ts

```diff
@@ -186,7 +186,7 @@ export function poolURLFor(
     return `https://app.xave.finance/#/pool`;
   }
 
-  return `${appUrl()}/pool/${poolId}`;
+  return `${appUrl()}/${getNetworkSlug(network)}/pool/${poolId}`;
```

Comment:
is there any difference in using `networkSlug` / `getNetworkSlug` ?

Reply:
`networkSlug` = slug for the current network the user is on
`getNetworkSlug` = function which returns a slug for any given network